cmd/create/machinepool/nodepool.go

```diff
@@ -189,6 +190,13 @@ func addNodePool(cmd *cobra.Command, clusterKey string, cluster *cmv1.Cluster, r
 		os.Exit(1)
 	}
 
-	r.Reporter.Infof("Machine pool '%s' created successfully on hosted cluster '%s'", createdNodePool.ID(), clusterKey)
-	r.Reporter.Infof("To view all machine pools, run 'rosa list machinepools -c %s'", clusterKey)
+	if output.HasFlag() {
+		if err = output.Print(createdNodePool); err != nil {
+			r.Reporter.Errorf("Unable to print node pool: %v", err)
```

Comment:
```suggestion
			r.Reporter.Errorf("Unable to print machine pool: %v", err)
```

To be consistent with message below this